.eslintrc

```diff
@@ -0,0 +1,52 @@
+{
+  "extends": [
+    "lisk-base",
+    "airbnb-base"
+  ],
```

Comment:
Did you consider extending [`airbnb`](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb) which also includes react-specific rules?